pkg/collector/corechecks/cluster/orchestrator/orchestrator.go

```diff
@@ -147,8 +147,9 @@ func (o *OrchestratorCheck) Configure(config, initConfig integration.Data, sourc
 
 	// check if cluster name is set
 	hostname, _ := coreutil.GetHostname()
-	if clusterName := clustername.GetClusterName(hostname); clusterName != "" {
-		o.orchestratorConfig.KubeClusterName = clusterName
+	o.orchestratorConfig.KubeClusterName := clustername.GetClusterName(hostname)
+	if o.orchestratorConfig.KubeClusterName == "" {
```

Comment:
Do we have some way to run unit tests for core check runs?

Reply:
I don't think there is any specific tooling around this yet, I tested the PR locally

```
2021-04-23 15:15:51 UTC | CLUSTER | ERROR | (pkg/collector/corechecks/loader.go:72 in Load) | core.loader: could not configure check orchestrator: orchestrator check is configured but the cluster name is empty
```

Reply:
👌 I thought we might have an easy way to add tests. If that not the case it is probably not worth it to invest that time 👍